intelmq/bots/outputs/elastic/output.py

```diff
@@ -54,15 +54,15 @@ def process(self):
         for field in self.flatten_fields:
             if field in event_dict:
                 val = event_dict[field]
-                # if it string try to convert to json
+                # if it's a string try to parse it as JSON
                 if isinstance(val, str):
                     try:
                         val = loads(val)
-                    except:
+                    except ValueError:
                         pass
                 if isinstance(val, Mapping):
                     for key, value in val.items():
-                        event_dict[key] = value
+                        event_dict[field + '_' + key] = value
```

Comment:
It's not only about overwriting, it's also the lost hierarchy, no valid harmonization, broken compatibility.

Once #793 is solved, this workaround can be removed anyway.